templates/features/env/database_cleaner.rb

```diff
@@ -0,0 +1,7 @@
+begin
+  DatabaseCleaner.strategy = :transaction
+rescue NameError
+  raise 'You need to add database_cleaner to your Gemfile (in the :test group) if you wish to use it.'
+end
+
+Cucumber::Rails::Database.javascript_strategy = :truncation
```

Comment:
By default, `rspec-rails` already puts this code into `features/env/custom_env.rb` as a comment. But I like having one file per concern instead of having one big custom config.